### PR TITLE
Fix GitHub Token permission

### PR DESCRIPTION
**Pull request by Tbaile**
Reverting the usage of a GitHub Token from the provided token in workflow to a secret

### CODE REVIEW

**Comment by Tbaile** [comment]
It appears that the `github.token` that the pipeline provides has permissions only on the repo that is running on. This is confirmed in [this paragraph](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret). A correct token needs to be set inside repo secrets named `github_token` (probably a @DavidePrincipi thing), then this can be safely merged.

File: .github/workflows/daily-check.yml [hunk 1]
```diff
@@ -10,4 +10,4 @@ jobs:
       - uses: actions/checkout@v3
       - run: bash check-new-release
         env:
-          GITHUBTOKEN: ${{ github.token }}
+          GITHUBTOKEN: ${{ secrets.github_token }}
```
Comment (DavidePrincipi) [inline .github/workflows/daily-check.yml]: May I suggest a name that recalls WHO owns the token?

```suggestion
          GITHUBTOKEN: ${{ secrets.nethbot_token }}
```

**Review by DavidePrincipi — COMMENTED** [review]
IIUC the scenario is similar to this one, right?

https://github.com/nethesis/tancredi/blob/master/.github/workflows/rpm.yml

**Comment by Tbaile** [comment]
> IIUC the scenario is similar to this one, right?
> 
> https://github.com/nethesis/tancredi/blob/master/.github/workflows/rpm.yml

Yes, it is, just renamed token in `nethbot_token`, merge when ready :smile:

**Pull request merged**